src/DataTable/__tests__/DataTable.test.js

```diff
@@ -545,6 +545,31 @@ describe('DataTable::progress/nodata', () => {
 
       expect(container.firstChild).toMatchSnapshot();
     });
+
+    test('should render correctly when persistTableHead and no data', () => {
+      const functionHadToBeClicked = jest.fn(() => true);
+      const onSortMock = jest.fn();
+
+      const mock = dataMock({
+        // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
+        name: (<div id="testColName" onClick={functionHadToBeClicked}>Test</div>),
```

Comment:
Why is the name block is needed or funcitonHadtoBeClicked? You just need to mock someone attempting to click the cell when data is 0 or prrogresspending is true

Reply:
these are the tests you need

```js
  test('should not call onSort if progressPending true', () => {
    const onSortMock = jest.fn();
    const mock = dataMock({ sortable: true });
    const { container } = render(
      <DataTable
        data={mock.data}
        columns={mock.columns}
        onSort={onSortMock}
        persistTableHead
        progressPending
      />,
    );

    fireEvent.click(container.querySelector('div[id="column-some.name"]'));

    expect(onSortMock).not.toBeCalled();
  });
```

and

```js
  test('should not call onSort if data is empty []', () => {
    const mock = dataMock({ sortable: true });
    const { container } = render(
      <DataTable
        data={[]}
        columns={mock.columns}
        onSort={onSortMock}
        persistTableHead
      />,
    );

    fireEvent.click(container.querySelector('div[id="column-some.name"]'));

    expect(onSortMock).not.toBeCalled();
  });
```

Reply:
No, i even want to be able to click on div or focus input in TableCol.

For example, here i cannot click on select under "Status" ...
![image](https://user-images.githubusercontent.com/6863446/99635251-febd6d00-2a41-11eb-9a31-431ce79c6e71.png)